EchoServer.UI/Greeting.cs

```diff
@@ -0,0 +1,12 @@
+namespace EchoServer;
+
+public class Greeting
+{
+    public string GetGreeting(int hour)
+    {
+        var greeting = hour < 12 ? "Good Morning!" :
+            hour < 17 ? "Good Afternoon!" : "Good Evening!";
+
+        return greeting;
+    }
```

Comment:
this is a fun algorithm.  I was expecting just a console print.  Why did you decide to go a more complicated route?

Reply:
Marie and I were just playing around/pairing! :-)

Reply:
Also gave you something to write more meaningful tests about!